x-pack/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -73,19 +66,19 @@ export class HeadlessChromiumDriverFactory {
 
       const chromiumArgs = this.getChromiumArgs(viewport);
 
-      let browser: Browser;
-      let page: Page;
+      let browser: puppeteer.Browser;
+      let page: puppeteer.Page;
       try {
-        browser = await puppeteerLaunch({
+        browser = await puppeteer.launch({
           pipe: !this.browserConfig.inspect,
           userDataDir: this.userDataDir,
           executablePath: this.binaryPath,
           ignoreHTTPSErrors: true,
           args: chromiumArgs,
           env: {
-            TZ: browserTimezone,
+            TZ: browserTimezone as string,
```

Comment:
I need to suss this one out a bit more, it could be that `tz` is always defined, but is marked optional in most places

Reply:
It's hard to test whether `TZ` actually works.. I have a feeling it might not always work.

Reply:
Yeah, this `env` param is just simple mechanic to inject env vars into chromium. I also think the `@types` for these are wrong: undefined is a totally valid value, and puppeteer doesn't throw errors if it is invalid.

I'm going to keep it this way for now since it doesn't seem to have any material effect on how jobs run, and I don't want to introduce any instability by making the `browserTimezone` parameter required in upstream methods.